base/tps/shared/conf/CS.cfg

```diff
@@ -13,10 +13,11 @@ applet._004=# Stock RSA,KeyRecover applet : 1.4.58768072.ijc
 applet._005=# RSA/KeyRecovery/GP211/SCP02, SCP03 applet : 1.5.558cdcff.ijc
 applet._006=# SCP03 AES server side keygen keywrap applet : 1.5.64260792.ijc
 applet._007=# Use GP211 applet only with SCP02 card
-applet._008=# For protocol > 1 do this ex: proto 3 : op.format.userKey.update.applet.requiredVersion.prot.3=1.5.558cdcff, 1.5.64260792.ijc for AES keywrap
+applet._008=# For protocol > 1 do this ex: proto 3 : op.format.userKey.update.applet.requiredVersion.prot.3=1.5.558cdcff, 1.5.64260792.ijc first for AES keywrap, latest for AES keywrap: 1.5.65cbf5a6.ijc : finall applet.
```

Comment:
I have trouble parsing this line of comment.  Would help if it could be written in more human-readable form.

Reply:
Will be addressed in next push.